gravitee-am-ui/src/app/domain/applications/application/advanced/factors/mfa-activate/conditional/mfa-conditional.component.html

```diff
@@ -36,32 +36,29 @@
     >
       <mat-icon>info_outline</mat-icon>
     </button>
-    <mat-hint>
-      <a href="https://documentation.gravitee.io/apim/guides/gravitee-expression-language" target="_blank">
-        <p class="link">Read more about Gravitee Expression Language</p>
-      </a>
-    </mat-hint>
   </mat-form-field>
-  <mat-slide-toggle (change)="switchSkipConditional()" [checked]="this.enrollment.enrollmentSkipActive"
-    >Allow users to skip Conditional enrollment
+  <a class="link" href="https://documentation.gravitee.io/apim/guides/gravitee-expression-language" target="_blank">
+    Read more about Gravitee Expression Language
+  </a>
+  <mat-slide-toggle style="margin-top: 30px" (change)="switchSkipConditional()" [checked]="this.enrollment.enrollmentSkipActive"
+    >Allow users to skip Conditional Enrollment
   </mat-slide-toggle>
-  <div *ngIf="this.enrollment.enrollmentSkipActive" style="padding-top: 20px">
+  <div fxLayout="column" *ngIf="this.enrollment.enrollmentSkipActive" style="margin-top: 30px">
     <p class="rule-title">Skip Conditional enrollment Rules</p>
     <mat-form-field appearance="outline" floatLabel="always">
       <input
         matInput
-        placeholder="Enter eligibility rule using Gravitee Expression Language."
+        placeholder="Skip MFA enrollment rules using Gravitee Expression language."
         name="skipEnrollmentConditionalMfaRulee"
         [(ngModel)]="this.enrollment.enrollmentSkipRule"
         (keyup)="updateSkipRule($event)"
       />
-      <mat-hint>
-        <a href="https://documentation.gravitee.io/apim/guides/gravitee-expression-language" target="_blank">
-          <p class="link">Read more about Gravitee Expression Language</p>
-        </a>
-      </mat-hint>
     </mat-form-field>
+    <a class="link" href="https://documentation.gravitee.io/apim/guides/gravitee-expression-language" target="_blank">
```

Comment:
please update this also in other components for consistency 
![image](https://github.com/gravitee-io/gravitee-access-management/assets/152597515/2c796d10-86c4-4026-a24a-58e1cae44f59)

Reply:
you probably didnt update the branch. its already updated.